docker/ci-base/Dockerfile

```diff
@@ -32,14 +32,19 @@ FROM ${CI_DOCKER_PROXY}node:14-buster
 
 RUN apt-get update && apt-get install -y \
     build-essential \
-    # required by cypress
+    # required by cypress and node
```

Comment:
btw (out of the scope of this). we don't need dependencies for cypress in the base image as cypress is run in another container. this is some legacy leftover.

Reply:
It would be nice to remove the comment and the libs that are not needed!